code/src/Album.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import {Artist} from './Artist'
+import './album.css'
+import './header.css'
+
+export const Album = (props) => {
+    console.log("props", props)
+    return (

```

Comment:
Looks like your editor is set up to do 4 spaces for indentation (should be 2). See [this question on stack overflow](https://stackoverflow.com/c/technigo/questions/253) for a bunch of things to check.